cmd/juju/controller/listmodels_test.go

```diff
@@ -245,18 +262,18 @@ func (s *ModelsSuite) TestModelsNonOwner(c *gc.C) {
 		"carlotta/test-model2         dummy         active      write   2015-03-01\n"+
 		"daiwik@external/test-model3  dummy         destroying  -       never connected\n"+
 		"\n")
+	s.checkAPICalls(c, "BestAPIVersion", "ListModels", "ModelInfo", "Close")
 }
 
 func (s *ModelsSuite) TestAllModels(c *gc.C) {
 	c.Assert(s.store.Models["fake"].Models, gc.HasLen, 0)
 	context, err := cmdtesting.RunCommand(c, s.newCommand(), "--all")
 	c.Assert(err, jc.ErrorIsNil)
-	c.Assert(s.api.all, jc.IsTrue)
 	c.Assert(cmdtesting.Stdout(context), gc.Equals, ""+
 		"Controller: fake\n"+
 		"\n"+
 		"Model                        Cloud/Region  Status      Access  Last connection\n"+
-		"admin/test-model1*           dummy         active      read    2015-03-20\n"+
+		"test-model1*                 dummy         active      read    2015-03-20\n"+
```

Comment:
Chased it down - it makes sense with respect to the changes on this branch: not really a change in behavior, more a correction to an incorrect behavior.
We have a logic to only present model name if (1) we know the current user and (2) this user happens to be the owner of the model.
In this case, we did not know (1) if the user has requested --all as the command argument. The code in this branch changed this path and now we *always* know the user for the command.
Since in this test, the user is 'admin', all models - where 'admin' is the owner - will now be displayed as a model name only and not fully qualified.